docs/conf.py

```diff
@@ -10,11 +10,39 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 
+import functools
+import json
 import os
 import sys
 from os import listdir
 from os.path import isdir, join
 
+import requests
+
+import semver
+
+
+def get_latest_tag(repo):
+    # TODO: Use releases/latest once we have a non-prelease
+    # https://developer.github.com/v3/repos/releases/#get-the-latest-release
+    names = []
+    url = "https://api.github.com/repos/" + repo + "/tags"
+
+    result = requests.get(url)
+
+    if result.status_code != 200:
+        print("Error getting latest tag: " + result.text)
+        return None
```

Comment:
It may be better to do `raise_for_status` and let the caller handle errors since we may run into other errors depending on the response (e.g. L38 below).